src/sentry/static/sentry/app/views/organizationEvents/events.jsx

```diff
@@ -167,14 +178,16 @@ class OrganizationEvents extends AsyncView {
               <RowDisplay>
                 {events.length ? t(`Results ${this.renderRowCounts()}`) : t('No Results')}
                 {!!events.length && (
-                  <TotalEventCount
-                    organization={organization}
-                    location={location}
-                    isAllResults={
-                      !parsedLinks.previous.results && !parsedLinks.next.results
-                    }
-                    numRows={events.length}
-                  />
+                  <Feature features={['internal-catchall']}>
```

Comment:
By moving the `Feature` component here, it ensures that `TotalEventCount` doesn't get mounted at all unless they have the required flags... previously it was being mounted and the API calls were being sent, but nothing would be displayed in the UI.